generator/wrapper_gen.py

```diff
@@ -175,7 +175,10 @@ def my_procid():
 
 def _get_data(x):
     if isinstance(x, pdDataFrame):
-        x = [x.loc[:,i].values for i in x]
+        if x._is_homogeneous_type:
```

Comment:
This seems be deprecated with pandas 1.1.0: https://github.com/pandas-dev/pandas/pull/30712
We should not rely on a deprecated and undocumented feature.